### PR TITLE
Search api

### PR DESCRIPTION
**Pull request by VECHAPAVANKUMAR**

### CODE REVIEW

File: Web Programming/project1/templates/userHome.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    </style>
+ 
+    <script>
+        
+        const email = "{{email}}"
+        
+        function search() {
+ 
+            const request = new XMLHttpRequest();
+            const value = document.querySelector('#search_item').value;
```
Comment (TulasiJagan49) [inline Web Programming/project1/templates/userHome.html]: @VECHAPAVANKUMAR Where are you passing this value to request??

Comment (VECHAPAVANKUMAR) [reply Web Programming/project1/templates/userHome.html]: I am passing this value at line number 92-93 along with email.

Comment (TulasiJagan49) [reply Web Programming/project1/templates/userHome.html]: @VECHAPAVANKUMAR Yep, got it. Thanks for the update.

**Review by TulasiJagan49 — APPROVED** [review]
@VECHAPAVANKUMAR You can merge into the master branch.

**Pull request merged**